src/app/backend/apihandler.go

```diff
@@ -74,6 +74,15 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 			Writes(Logs{}))
 	wsContainer.Add(logsWs)
 
+	eventsWs := new(restful.WebService)
+	eventsWs.Path("/api/events").
+		Produces(restful.MIME_JSON)
+	eventsWs.Route(
+		eventsWs.GET("/{namespace}").
```

Comment:
Why do we need only namespace? How about pod name? What does this WS actually return? Can you document it?